beancount/core/data.py

```diff
@@ -225,9 +225,9 @@ class Transaction(NamedTuple):
 # Attributes:
 #   txn: The parent Transaction instance.
 #   posting: The Posting instance.
-TxnPosting = NamedTuple('TxnPosting', [
-    ('txn', Transaction),
-    ('posting', Posting)])
+class TxnPosting( NamedTuple ):
```

Comment:
Does not matter much, but there are extra spaces within the parentheses.

Reply:
Thanks